src/walletSdk/index.ts

```diff
@@ -97,13 +100,13 @@ export class StellarConfiguration {
 
 export class ApplicationConfiguration {
   defaultSigner: WalletSigner;
-  defaultClient: HttpClient;
-  constructor(
-    defaultSigner: WalletSigner = DefaultSigner,
-    defaultCliengConfig: ClientConfig = {}
-  ) {
+  defaultClient: AxiosInstance;
+  constructor(defaultSigner: WalletSigner = DefaultSigner) {
     this.defaultSigner = defaultSigner;
-    // TODO - default client
-    this.defaultClient = new HttpClient();
+    this.defaultClient = axios.create({
```

Comment:
@Ifropc regarding your comment from https://github.com/stellar/typescript-wallet-sdk/pull/5#discussion_r1189200628

using axios which allows http by default

Reply:
Looking good, thanks!
About the comment above, the idea is that when developer creates anchor like this
`.anchor(my-domain.com)` it defaults to `https://my-domain.com` internally.
However, we want to be able to also use `http://my-domain.com`. In kt wsdk, developer can either create anchor implicitly via `.anchor(http://my-domain.com)` or simply setting default schema to http.
Can we add test, that simple stuff with http schema works? (Not sure if demo wallet allows http schema, but we should at least check that we are making requests to http in underlying classes, such as Auth)
So, simply testing Auth with http should be good. 
(We can merge this PR and create a test in a separate PR)

Reply:
ah I see what you're saying. I added this test for calling w/ http, lmk what you think (it was quick so included in this PR :) )
https://github.com/stellar/typescript-wallet-sdk/pull/8/commits/a5a817ca7d5d9e49a86fd56100700bfd8bd9ed29

Reply:
LGTM